run/django/requirements.txt

```
@@ -1,6 +1,6 @@
-Django==5.1.6; python_version >= "3.10"
-Django==4.2.19; python_version >= "3.8" and python_version < "3.10"
-Django==3.2.25; python_version < "3.8"
+Django==5.1.7; python_version >= "3.10"
+Django==5.1.7; python_version >= "3.8" and python_version < "3.10"
+Django==5.1.7; python_version < "3.8"
```

```suggestion
Django==5.1.7; python_version >= "3.10"
Django==4.2.20; python_version >= "3.8" and python_version < "3.10"
```

We're testing 3.9+ here, so let's drop the 3.8 lock